tasks/sep/base-005-fp-holocene-upgrade/VALIDATION.md

```diff
@@ -0,0 +1,31 @@
+# Validation
+
+This document can be used to validate the state diff resulting from the execution of the upgrade
+transaction.
+
+For each contract listed in the state diff, please verify that no contracts or state changes shown in the Tenderly diff are missing from this document. Additionally, please verify that for each contract:
+
+- The following state changes (and none others) are made to that contract. This validates that no unexpected state changes occur.
+- All addresses (in section headers and storage values) match the provided name, using the Etherscan and Superchain Registry links provided. This validates the bytecode deployed at the addresses contains the correct logic.
+- All key values match the semantic meaning provided, which can be validated using the storage layout links provided.
+
+## State Changes
+
+### `0x0fe884546476dDd290eC46318785046ef68a0BA9` (`GnosisSafeProxy`)
+
+- **Key**: `0x0000000000000000000000000000000000000000000000000000000000000005` <br/>
+  **Before**: `0x0000000000000000000000000000000000000000000000000000000000000009` <br/>
+  **After**: `0x000000000000000000000000000000000000000000000000000000000000000a` <br/>
+  **Meaning**: The nonce in slot `0x05` of the L1 Gnosis Safe `ProxyAdmin` owner is incremented from 9 to 10.
+
+### `0xd6E6dBf4F7EA0ac412fD8b65ED297e64BB7a06E1` (`DisputeGameFactoryProxy`)
+
+- **Key**: `0xffdfc1249c027f9191656349feb0761381bb32c9f557e01f419fd08754bf5a1b` <br/>
+  **Before**: `0x0000000000000000000000005062792ed6a85cf72a1424a1b7f39ed0f7972a4b` <br/>
+  **After**: `0x000000000000000000000000b7fb44a61fde2b9db28a84366e168b14d1a1b103` <br/>
+  **Meaning**: Updates the CANNON game type implementation. Verify that the new implementation is set using `cast call 0xd6E6dBf4F7EA0ac412fD8b65ED297e64BB7a06E1 "gameImpls(uint32)(address)" 0`. Where `0` is the [`CANNON` game type](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.4.0/packages/contracts-bedrock/src/dispute/lib/Types.sol#L28).
```

Comment:
For this as well as the next key/value pair, the **old** implementation value is derived from the output of `cast call`. This should be cleaned up before merging

Reply:
Added clarification to these paragraphs